src/apps/tco2_dashboard/app.py

```diff
@@ -450,32 +451,28 @@ def get_mco2_data():
     return df_bridged, df_bridged_tx, df_retired, df_moss_retired
 
 
+def fallback_verra():
+    fallback_note = VERRA_FALLBACK_NOTE
+    df_verra = pd.read_csv(VERRA_FALLBACK_URL)
+    df_verra = df_verra[verra_columns]
+    return df_verra, fallback_note
+
+
 @cache.memoize()
 def get_verra_data():
+    debug("Compute: get_verra_data")
     use_fallback_data = False
     if use_fallback_data:
-        fallback_note = VERRA_FALLBACK_NOTE
-        df_verra = pd.read_csv(VERRA_FALLBACK_URL)
-        df_verra = df_verra[verra_columns]
+        return fallback_verra()
     else:
-        try:
-            fallback_note = ""
-            r = requests.post(
-                "https://registry.verra.org/uiapi/asset/asset/search?$maxResults=2000&$count=true&$skip=0&format=csv",
-                json={"program": "VCS", "issuanceTypeCodes": ["ISSUE"]},
-            )
-            df_verra = pd.DataFrame(r.json()["value"]).rename(columns=verra_rename_map)
-        except requests.exceptions.RequestException as err:
-            print(err)
-            fallback_note = VERRA_FALLBACK_NOTE
-            df_verra = pd.read_csv(VERRA_FALLBACK_URL)
-            df_verra = df_verra[verra_columns]
-    return df_verra, fallback_note
+        fallback_note = ""
+        df_verra = load_s3_data("raw_verra_data")
+        return df_verra, fallback_note
```

Comment:
I assume S3 should be hyper reliable and it is better to crash if there is an issue.